### PR TITLE
GUI: fix invalid unicode

### PR DESCRIPTION
**Pull request by berniev**
Thank you for creating a pull request to contribute to FreeCAD! Place an "X" in between the brackets below to "check off" to confirm that you have satisfied the requirement, or ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10) if there is something you don't understand.

- [ ]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

Please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [1.0 Changelog Forum Thread](https://forum.freecad.org/viewtopic.php?f=10&t=69438).

---
Triggered from build warning

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7901">pipeline status</a> for feature branch [PR_7901](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7901). Pipeline [707603884](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/707603884) was triggered at [f3f916a](https://github.com/FreeCAD/FreeCAD/pull/7901/commits/f3f916a010704505ad230ef5e2e2aef588625cda). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Gui/TreeItemMode.h [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /***************************************************************************
- *   Copyright (c) 2004 J�rgen Riegel <juergen.riegel@web.de>              *
+ *   Copyright (c) 2004 Jürgen Riegel <juergen.riegel@web.de>              *
```
Comment (luzpaz) [inline src/Gui/TreeItemMode.h:2]: How is it invalid? + the asterisk at the end of the line needs to be fixed as well

Comment (berniev) [reply src/Gui/TreeItemMode.h:2]: See how second letter in Juergen's first name is '?'.  I don't know preciseely what is wrong with it, but throws build warnings, and also shows '?' in my IDE.

Comment (luzpaz) [reply src/Gui/TreeItemMode.h:2]: I don't see it in vscode or on github.

Comment (luzpaz) [reply src/Gui/TreeItemMode.h:2]: Screenshot_20221130_065512

**Comment by berniev** [comment]
Screen Shot 2022-12-01 at 4 22 25 am

Something weird going on at your end perhaps?

**Comment by berniev** [comment]
vscode:
Screen Shot 2022-12-01 at 4 34 13 am

**Comment by luzpaz** [comment]
Lets just change it to `Juergen` since it's creating issues?

**Comment by berniev** [comment]
There are already 801 occurrences of Jürgen (with the umlaut) in FC, so an umlaut is not normally an issue. 

IMO all the non-umlaut ones should be updated -- for uniformity, and out of respect for the man. 

I'm curious why you are having issues. Could you look at e.g. src/App/Application.cpp and see how the umlaut shows for you?

**Comment by luzpaz** [comment]
I once updated all the umlauts and what happened was that the python files reported errors because they required  an extra line at the top for unicode purposes. 

Here's what my idea is showing:  
Screenshot_20221201_055215

**Comment by berniev** [comment]
@luzpaz You don't mention what that image is of, exactly. Pls advise.

**Comment by luzpaz** [comment]
Sorry, the file is `src/Gui/TreeItemMode.h` as it now before your PR.

**Comment by berniev** [comment]
Yes! That is what I see too: the question mark indicates invalid UTF-8. This is what prompted the PR, never mind the invalid UTF-8 causes build failure (I am using C++20, if it matters). So maybe just merge and be done? BTW What changed at your end?

**Review by luzpaz — APPROVED** [review]

**Pull request merged**